<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedSearchDialog</class>
 <widget class="QDialog" name="AdvancedSearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HexWalk - Advanced Find</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,5">
     <item>
      <widget class="QGroupBox" name="gbFind">
       <property name="title">
        <string>Find</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="cbFindFormat">
          <item>
           <property name="text">
            <string>UTF-8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UTF-16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hex</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbFind">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbOptions">
       <property name="title">
        <string>Options</string>
       </property>
       <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,1" columnstretch="0,0" rowminimumheight="0,0,0,1" columnminimumwidth="0,0">
        <item row="3" column="0">
         <widget class="QCheckBox" name="cbBackwards">
          <property name="text">
           <string>&amp;Backwards</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="cbBegin">
          <property name="text">
           <string>Start from beginning</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="cbRegex">
          <property name="text">
           <string>Regex</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="cbCase">
          <property name="text">
           <string>Case Insensitive</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="cbInvertMatch">
          <property name="text">
           <string>Invert Match</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbResults">
       <property name="title">
        <string>Results</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="resultsTableView">
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pbFind">
       <property name="text">
        <string>&amp;Find</string>
       </property>
       <property name="shortcut">
        <string>F3</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbFindAll">
       <property name="text">
        <string>Find All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbFind</tabstop>
  <tabstop>cbFindFormat</tabstop>
  <tabstop>cbBackwards</tabstop>
  <tabstop>pbFind</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>AdvancedSearchDialog</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
